client/scripts/helpers/calculator_helper.ts

```diff
@@ -0,0 +1,144 @@
+import moment from 'moment-twitter'
+import {AssetInfo} from '../models/AssetInfo'
+
+// duplicated in helpers.ts
```

Comment:
if these are duplicated can you please update/use the helper functions there?

Reply:
Duplicated only because number.toFixed(2) change was required to have better UI